src/main/java/woocommerce/utils/DateTimeUtils.java

```diff
@@ -1,16 +1,20 @@
 package woocommerce.utils;
 
 import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.Date;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
 
 public class DateTimeUtils {
-    private static final SimpleDateFormat ISO_8601 = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm");
+    private static final DateTimeFormatter ISO_8601 = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm");
```

Comment:
Woud you please add second to this format as we have discussed before?
Thanks a lot